doc/integrator/fulltext_search.rst

```diff
@@ -289,3 +289,33 @@ And define the configuration in the ``vars.yaml`` file:
 
 ``fr: fr`` is a link between the pyramid language and the text search configuration, by default it is
 ``fr: french`` because the default french text search configuration is named 'french'.
+
+
+Don't use Synonym and Thesaurus Dictionary
+------------------------------------------
+
+With those dictionaries e.-g. 'alignement' is simplifyed as 'align':
+
+.. code:: sql
+
+   SELECT to_tsvector('fr', 'alignement');
+   'align':1
+
+Then 'alignem' don't match with it:
```

Comment:
Thus, 'alignem' does not match in the search, which is unexpected behavior for the user:

Reply:
New:

Thus, 'alignem' does not match in the search, which might be considered unexpected behavior by users: